server/compaction-coordinator/src/test/java/org/apache/accumulo/coordinator/CompactionCoordinatorTest.java

```diff
@@ -268,7 +269,8 @@ public void testCoordinatorColdStartNoCompactions() throws Exception {
         .andReturn(runningCompactions);
     expect(ExternalCompactionUtil.getCompactorAddrs(context)).andReturn(Map.of()).anyTimes();
 
-    CompactionFinalizer finalizer = PowerMock.createNiceMock(CompactionFinalizer.class);
+    DefaultCompactionFinalizer finalizer =
+        PowerMock.createNiceMock(DefaultCompactionFinalizer.class);
```

Comment:
Just a heads up: PowerMock was removed entirely from the main branch and this test was substantially reworked in the main branch. I'm not sure if that's something you might want to backport to update this test, or if it will be easy enough to update the test in the main branch.